srcpkgs/singularity/template

```diff
@@ -0,0 +1,58 @@
+# Template file for 'singularity'
+pkgname=singularity
+version=3.7.1
+revision=1
+wrksrc="${pkgname}"
+build_style=go
+go_import_path=github.com/sylabs.io/singularity
+go_mod_mode=vendor
+hostmakedepends="wget pkg-config go cryptsetup"
+makedepends="libressl-devel libseccomp-devel"
+depends="squashfs-tools cryptsetup"
+short_desc="HPC centric container platform"
+maintainer="Olaf Mersmann <olafm@p-value.net>"
+license="BSD-3-Clause-LBNL"
+homepage="https://sylabs.io/singularity/"
+distfiles="https://github.com/hpcng/${pkgname}/releases/download/v${version}/${pkgname}-${version}.tar.gz"
+checksum=82d2c65063560195ec34551931be3c325b95e8e2009e92755fd7daad346e083c
+nocross=yes
+
+do_configure() {
+	./mconfig \
+		--prefix=/usr \
+		--exec-prefix=/usr \
+		--bindir=/usr/bin \
+		--libexecdir=/usr/libexec \
+		--sysconfdir=/etc \
+		--sharedstatedir=/var/lib \
+		--mandir=/usr/share/man \
+		--localstatedir=/var/lib
+
+	# Fixup Makefile
+	# * Force use of vendored packages
+	# * Don't install bash completions into /etc/bash_completion.d
+	vsed -i \
+		-e 's@^GO_MODFLAGS := .*@GO_MODFLAGS := -mod=vendor@' \
+		-e 's@INSTALLFILES += $(bash_completion_INSTALL)@@' \
+		builddir/Makefile
+}
+
+do_build() {
+	export srcdir=$PWD
+	make -C builddir old_config=
+}
+
+do_install() {
+	make DESTDIR="${DESTDIR}" -C builddir install man
+	vlicense LICENSE.md
+	vcompletion builddir/etc/bash_completion.d/singularity bash
+	rm -fr "${DESTDIR}"/usr/lib64
```

Comment:
Given that `/usr/lib64` should be a symlink to `/usr/lib`, this seems very wrong... Why is it required?

Also, I prefer to use `-r` so it errors out if the file doesn't exist, which would mean the command isn't necessary :)

Reply:
Can't remember why I added it, it's not necessary so I removed it.